conf/export_test.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright © 2019, 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conf
+
+// Export for testing
+//
+// Please look into the following blogpost:
+// https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd
+// to see why this trick is needed.
+var (
+	CheckIfFileExists = checkIfFileExists
+)
```

Comment:
I have a question about this kind of files. If somebody imports the package `conf`, will it include this definition that makes one "private" function public? Or the `_test` files are ignored when compiling the `package` for some reason?

Reply:
The second is true - there files are ignored during compilation. This is easy to prove:

```
$ nm insights-content-service |grep checkIfFileExists
000000000084d980 T github.com/RedHatInsights/insights-content-service/conf.checkIfFileExists
00000000009dee90 R github.com/RedHatInsights/insights-content-service/conf.checkIfFileExists.stkobj

$ nm insights-content-service |grep CheckIfFileExists

```

Reply:
Nice! TYVM